fe/fe-core/src/main/java/org/apache/doris/nereids/operators/plans/physical/PhysicalHeapSort.java

```diff
@@ -32,37 +32,29 @@
 /**
  * Physical sort plan operator.
  */
-public class PhysicalSort extends PhysicalUnaryOperator {
-
+public class PhysicalHeapSort extends PhysicalUnaryOperator {
+    // Default offset is 0.
     private final int offset;
 
-    private final int limit;
-
-    private final List<OrderKey> orderList;
-
+    private final List<OrderKey> orderKeys;
     private final boolean useTopN;
```

Comment:
useTopN need to be removed. In translator, HeapSort will translate to topN and MergeSort will translate to normal sort